src/client/kinto_http.rs

```diff
@@ -142,7 +143,15 @@ pub fn get_changeset(
         .map_or_else(|| -1, |v| v.parse().unwrap_or(-1));
 
     debug!("Download {:?} bytes...", size);
-    Ok(response.json::<ChangesetResponse>()?)
+    let mut changeset: ChangesetResponse = response.json()?;
+
+    // Check if server is indicating to clients to back-off.
+    changeset.backoff = response
+        .headers
+        .get("backoff")
+        .map_or_else(|| None, |v| v.parse().ok());
```

Comment:
This can be simplified to
```suggestion
    changeset.backoff = response.headers.get("backoff").and_then(|v| v.parse().ok());
```
Note that we are suppressing any parsing errors, but I think that's fine.